cosi.proto

```diff
@@ -203,27 +195,20 @@ message ProvisionerDeleteBucketResponse {
 
 message ProvisionerGrantBucketAccessRequest {
     // This field is REQUIRED
-    // Protocol specific information required by the call is passed in as key,value pairs.
-    Protocol protocol = 1;
-
-    // This field is OPTIONAL
-    // Protocol specific information required by the call is passed in as key,value pairs.
-    map<string,string> parameters = 2;
-
-    // This field is OPTIONAL
-    // Principal for which access is requested
-    string principal = 3;
+    // BucketName is a globally unique identifier for the bucket
+	// in the object storage provider 
+	string bucketID = 1;
 
     // This field is REQUIRED
     // Requested Access policy, ex: {"Effect":"Allow","Action":"s3:PutObject","Resource":"arn:aws:s3:::profilepics/*"}
-    string access_policy = 4;
+    string access_policy = 2;
 }
 
 message ProvisionerGrantBucketAccessResponse {
     // This field is OPTIONAL
-    // This is the account that is being provided access. This will
+    // This is the accountID that is being provided access. This will
     // be required later to revoke access. 
-    string principal = 1;
+    string accountID = 1;
```

Comment:
accountID seems more intuitive and also is symmetric with bucketID

Reply:
Ack. I don't have the context on this, maybe worth having someone else who was part of that discussion take a look